src/os_win32.c

```diff
@@ -5438,25 +5508,30 @@ mch_call_shell(
     int		x = 0;
     int		tmode = cur_tmode;
     WCHAR	szShellTitle[512];
+    size_t	szShellTitlelen;
 
 #ifdef FEAT_EVAL
     ch_log(NULL, "executing shell command: %s", cmd);
 #endif
     // Change the title to reflect that we are in a subshell.
-    if (GetConsoleTitleW(szShellTitle, ARRAY_LENGTH(szShellTitle) - 4) > 0)
+    szShellTitlelen = (size_t)GetConsoleTitleW(szShellTitle, sizeof(szShellTitle) - 4);
```

Comment:
`szShellTitle` is an array of WCHAR. Using `sizeof` is not correct:
```suggestion
    szShellTitlelen = (size_t)GetConsoleTitleW(szShellTitle, ARRAY_LENGTH(szShellTitle) - 4);
```

Reply:
Thanks for the tip!